src/components/InstructorsOnly/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
+import {Error} from 'egghead-ui'
 import {instructorsOnlyDescriptionText} from 'utils/text'
 import Main from 'components/Main'
-import {Error} from 'egghead-ui'
```

Comment:
I moved all egghead-ui imports next to the other node_modules at the top of each file